docs/pr-review-guidelines.md

```
@@ -0,0 +1,75 @@
+## Initial Checks
+
+- [ ] **Is it a code PR?** If yes, do they have an issue (no matter how small), linked and attached to the PR, describing the desired change or issue?
+  > As it stands today, we don't have strict rules for non code changing PRs – such as README updates or doc changes
+
+- [ ] **Our PR CI passes**
+
+- [ ] **Have they answered the [PR template](../.github/pull_request_template.md) properly** – provided screenshots, have sane responses to the questions, etc
+
+- [ ] **Does it have tests?** (Refer to [Testing](testing.md) documentation)
+
+   - [ ] **Have you made sure to test the PR image on your cluster?** (this is critically important if there are permission changes since we only have cluster-admin locally; so if there are permission changes along with it, test with and without those permissions to verify the flow on-cluster)
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 3

(MD007, ul-indent)